Common/Source/Sound/ExtDev/Sound.cpp

```diff
@@ -107,12 +128,24 @@ void PlayResource (const TCHAR* lpName) {
     const tstring& nmeaStr = _sound_table.getNmeaStr(sound_code);
 
     if (!nmeaStr.empty()) {
-        if (UseExtSound1) {
+        if (UseExtSound[0]) {
             devWriteNMEAString(devA(), nmeaStr.c_str());
         }
-        if (UseExtSound2) {
+        if (UseExtSound[1]) {
             devWriteNMEAString(devB(), nmeaStr.c_str());
         }
+        if (UseExtSound[2]) {
+            devWriteNMEAString(devC(), nmeaStr.c_str());
+        }
+        if (UseExtSound[3]) {
+            devWriteNMEAString(devD(), nmeaStr.c_str());
+        }
+        if (UseExtSound[4]) {
+            devWriteNMEAString(devE(), nmeaStr.c_str());
+        }
+        if (UseExtSound[5]) {
+            devWriteNMEAString(devF(), nmeaStr.c_str());
+        }
     }
```

Comment:
line 128 to 149 duplicate line 84 to 107 -> make function for avoid mistake in futur ...